operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
-  * Use introductions as learning opportunities to understand different parts of the company \(ask questions!\) 
+* Introductions to team members through 1-1s
+  * Use introductions as learning opportunities to understand different parts of the company \(ask questions!\)
 * Introduction to product
-* Introduction to our “Fast, Obvious, Forgiving” design principles 
+* Introduction to our “Fast, Obvious, Forgiving” design principles
 * Introduction to key resources
 * Make your first contribution on GitHub
 
 Activities:
 
 Customer Obsession:
 
-* Join and introduce yourself in these channels, ask to join any and all customer calls - see if you can get a couple scheduled this week 
-  * [CSM:PM Engagement](https://community-daily.mattermost.com/private-core/channels/csmpm-engagement) \(Stu, Emily, Rosie, Jenn\)
-  * [Solution Architects](https://community-daily.mattermost.com/private-core/channels/solutions-architects)
-  * [Pre-Sales Support](https://community-daily.mattermost.com/private-core/channels/pre-sales-support)
-* Join and review the topics and posts in these channels to get a better understanding of our customers 
-  * [Customer Feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
-  * [Customer Support](https://community-daily.mattermost.com/private-core/channels/community)
+* Join and introduce yourself in these channels, ask to join any and all customer calls - see if you can get a couple scheduled this week:
+  * [CSM:PM Engagement](https://community.mattermost.com/private-core/channels/csmpm-engagement) \(Stu, Emily, Rosie, Jenn\)
+  * [Solution Architects](https://community.mattermost.com/private-core/channels/solutions-architects)
+  * [Pre-Sales Support](https://community.mattermost.com/private-core/channels/pre-sales-support)
+* Join and review the topics and posts in these channels to get a better understanding of our customers
+  * [Customer Feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
+  * [Customer Support](https://community.mattermost.com/private-core/channels/community)
 * Begin watching training videos on [academy.mattermost.com ](https://academy.mattermost.com/)
 * Begin reading [docs.mattermost.com](https://docs.mattermost.com/)
 
 Team:
 
-* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings 
-  * Ask questions to understand development, design, QA processes 
-* Meet with Amy \(Release Manager\) and read the release process doc 
-* Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\) - This will be scheduled for you 
+* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings
+  * Ask questions to understand development, design, QA processes
+* Meet with Amy \(Release Manager\) and read the release process doc
+* Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\). This will be scheduled for you
```

Comment:
```suggestion
* Meet with Ian \([https://github.com/it33/readme/blob/master/README.md](https://github.com/it33/readme/blob/master/README.md/)\). This will be scheduled for you
```